docs/jp/coding-guidelines.md

```
@@ -0,0 +1,7 @@
+# コーディングガイドライン
+
+> English version: [../coding-guidelines.md](../coding-guidelines.md)
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)